book/animation/animated/more-animated.md

```diff
@@ -0,0 +1,87 @@
+# More Animated!
+
+The Animated library allows us to do a lot more things such as composing animations or driving animations through gestures and input events. Let's look at a few examples.
+
+
+
+## Interpolation
+
+Using the same `Animated.Value` you can drive multiple animations through interpolation. Interpolation can also be used to convert numbers into `rgb` colours or rotation values in `degrees`.
+
+```js
+this.state = { animatedVal: new Animated.Value(0) };
+
+const marginLeftAnimation = this.state.animatedValue.interpolate({
+  inputRange: [-300, -100, 0, 100, 101],
+  outputRange: [300,    0, 1,   0,   0],
```

Comment:
This is a little confusing. I might add the example of the resulting map that exists over on https://facebook.github.io/react-native/docs/animations.html